irohad/network/impl/peer_communication_service_impl.cpp

```diff
@@ -27,8 +27,9 @@ namespace iroha {
     bool PeerCommunicationServiceImpl::propagate_batch(
         std::shared_ptr<shared_model::interface::TransactionBatch> batch)
         const {
-      log_->info("propagate batch");
-      return ordering_gate_->propagateBatch(batch);
+      const bool is_propagated = ordering_gate_->propagateBatch(batch);
+      log_->info("{}propagated batch", (is_propagated ? "" : "not "));
```

Comment:
it is OK. probably a bit pretentiously but ok.

Why not just?) 
```suggestion
      log_->info("batch propagation result: {}", is_propagated);
```

Reply:
I see the original variant more accurate, as "true" and "false" are not the results of propagation.